packages/components/src/wrap.js

```diff
@@ -35,22 +44,23 @@ export default function wrap(Component, key) {
 			</Component>
 		);
 	};
-	Object.keys(Component).forEach(attr => {
-		Wrapper[attr] = Component[attr];
-	});
+	Object.keys(Component)
```

Comment:
would be nice to use reduce to limit the number of iteration
would be nice to add some form of memoize to limit the number of execution

non blocking comment could be nice to add as a comment to the code because if im not mistaken it is executed A LOT

Reply:
This is executed once at bootstrap to expose bootstrap components and the result is exported.